.github/workflows/build-deploy.yml

```diff
@@ -26,8 +27,12 @@ jobs:
   #         version: v1.60.3
   #         args: --timeout 600s --verbose ./...
 
-  test:
+  Test-Operator:
```

Comment:
This job now uses a different naming pattern than the rest - might be nice to have everything consistent, but I'm also ok with it if you'd like to keep it that way

Reply:
Thx, changed it back to test.